.github/workflows/ubuntu.yml

```diff
@@ -4,7 +4,6 @@ on:
   pull_request:
     branches: [ gcos4gnucobol-3.x ]
   push:
-    branches: [ gcos4gnucobol-3.x ]
```

Comment:
Don't we want to keep that ? (when merging upstream changes from `gnucobol-3.x`, we don't go through a PR)

Reply:
Maybe we should PR the merges. Here it's only removing a restriction, the CI will run on *any* push, not just a push on that branch.